src/jit/codegenxarch.cpp

```diff
@@ -1557,7 +1557,7 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
 
             GenTreePtr op1 = treeNode->gtGetOp1();
             genConsumeRegs(op1);
-            emit->emitInsBinary(ins_Store(targetType), emitTypeSize(treeNode), treeNode, op1);
```

Comment:
Same comment here as below: even though this is a store to the stack, it is a store to an on-stack field and must therefore be appropriately-sized.